### ISSUE TITLE
Running tests adds untracked files

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/augur#863]
### Current Behavior
Running tests from a clean git state produces untracked files

### Expected behavior
No untracked files

### How to reproduce
Steps to reproduce the current behavior:
1. `bash run_tests.sh`
2. `git status`
```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
        tests/functional/tree/full_aligned-delim.fasta.contree
        tests/functional/tree/full_aligned-delim.fasta.splits.nex
        tests/functional/tree/full_aligned-delim.fasta.ufboot
        tests/functional/tree/masked_aligned-delim.iqtree.log.xz
        tests/functional/tree/masked_aligned.fasta.xz
```

### Possible solution
(optional)

### Your environment
Python 3.6.12, Augur at commit e4fdf59f

**Comment by victorlin** [comment]
Solution: have tests clean up or run in tmp directory

**Comment by victorlin** [comment]
Reopening since #863 only addressed untracked files for filter, not tree as shown in the issue description. This can be closed when https://github.com/nextstrain/augur/issues/863 is closed.

**Comment by victorlin** [comment]
Oops I misread the history. This was properly closed in #1077
